Dockerfile.local

```diff
@@ -1,18 +1,6 @@
-FROM	ubuntu:20.04
+ARG	FROM_PARENT
+FROM	${FROM_PARENT}
```

Comment:
it could be part of the main `Dockerfile` as a separate target (something like `--target=local`?). will keep dependabot happy too!

Reply:
Done!